tests/integration/double_vote.go

```diff
@@ -159,6 +159,9 @@ func (s *CCVTestSuite) TestHandleConsumerDoubleVoting() {
 	consuAddr := types.NewConsumerConsAddress(sdk.ConsAddress(consuVal.Address.Bytes()))
 	provAddr := s.providerApp.GetProviderKeeper().GetProviderAddrFromConsumerAddr(s.providerCtx(), s.consumerChain.ChainID, consuAddr)
 
+	validator, _ := s.providerApp.GetTestStakingKeeper().GetValidator(s.providerCtx(), provAddr.ToSdkConsAddr().Bytes())
```

Comment:
This is a simple test that some type of slashing takes place when we call `HandleConsumerDoubleVoting`.

Ideally, I would also like to verify that undelegations and redelegations are slashed as well with something similar to:
```
bondAmt := sdk.NewInt(10000)
delAddr := s.providerChain.SenderAccount.GetAddress()
_, _ = delegateAndUndelegate(s, delAddr, bondAmt, int64(20))

unbs := s.providerApp.GetTestStakingKeeper().GetUnbondingDelegationsFromValidator(provCtx, valAddress)
// check `unbs` have less tokens
```

However, I couldn't find a way to "clean" undelegations/redelegations for the next test case inside the `for _, tc := range testCases {` loop and this confused things a bit as we traverse through the test cases. I'll probably break the tests into 3 tests where i) all tests that have `expPass == false`, ii) "valid double voting evidence 1 - should pass", and iii) "valid double voting evidence 2 - should pass".